test-e2e/roles/pvc_affinity_pod/templates/pod.yml.j2

```diff
@@ -13,11 +13,7 @@ spec:
       command: ["sh", "-c"]
       args:
         - |
-          set -x -e -o pipefail
-
-          mkdir -p `dirname /mnt/{{ path }}`
-          echo '{{ data }}' > '/mnt/{{ path }}'
-          sync
+          sleep 99999
```

Comment:
This Pod should be able to immediately exit. Once the volume has been provisioned, it's zone is locked.

Reply:
I just may not be understanding this all correctly - but I thought the idea behind leaving the pod running was to force later replicationsource/destintations that use direct to pull the node from that running pod.  In this way we confirm that the nodeSelector is working on the subsequent job?

Reply:
:man_facepalming: Nope, I missed the point. I thought this role was for PVC provisioning as opposed to testing VolSync's RWO affinity.